client/src/modules/privacy-policy/components/TermsOfUse.en.md

```diff
@@ -0,0 +1,40 @@
+# Sandstream Terms of Service
+
+## 1\. Terms
+
+By accessing the website at [eoc.app.sanddev.com](https://eoc.app.sanddev.com), you are agreeing to be bound by these terms of service, all applicable laws and regulations, and agree that you are responsible for compliance with any applicable local laws. If you do not agree with any of these terms, you are prohibited from using or accessing this site. The materials contained in this website are protected by applicable copyright and trademark law.
+
+## 2\. Use License
```

Comment:
how does it comply to MIT?

Reply:
I was thinking about it and I'm not sure. Every generator or page which helps writing such content generates/shows a restrictive license statement. I'll try to change it and look for other examples

Reply:
This license is not compatible with MIT. 
MIT allows us to do everything with the software, provides no warranty, etc. 
As this license claims, users can have temporarily only one copy of this software and there are 5 points about what users can't do. MIT allows users to do everything with the software. 

In my opinion, we should use MIT license here if we want to distribute EOC under MIT and be consistent with our assumptions from the beginning of the project.

Reply:
I changed the license statement but: MIT license applies to the software, code. Of course, users (developers) can do everything with the software (code) and we provide no warranty for this. But here we have deployed and a full-working web application where Sandstream is the owner and administrator. So I think we have to write certain rules that apply to application users (not developers who can use the code any way they like).

Reply:
😀 😁 😂
https://www.google.com/search?q=prawnik+wroc%C5%82aw&rlz=1C5CHFA_enPL870PL870&oq=prawnik+wroc%C5%82aw&aqs=chrome..69i57j0l5.2465j0j7&sourceid=chrome&ie=UTF-8